File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.java
class Solution {
    public boolean numberHasEvenDigit(int num){
        int digitCount = 0;
        while(num!=0){
            num= num/10;
            digitCount++;
        }
        return digitCount%2==0;
    }
    public int findNumbers(int[] nums) {
        int EvenCount = 0;
        for(int i=0;i<nums.length;i++){
            if(numberHasEvenDigit(nums[i])){
                EvenCount++;
            }
            
        }
        return EvenCount;
        
    }
}

 

File: 1295-find-numbers-with-even-number-of-digits/1295-find-numbers-with-even-number-of-digits.py
class Solution(object):

    def numberHasEvenDigit(self, num):
        digit_count = 0

        while num != 0:
            digit_count += 1
            num = num // 10

        return digit_count % 2 == 0

    def findNumbers(self, nums):
        even_count = 0

        for i in range(len(nums)):
            if self.numberHasEvenDigit(nums[i]):
                even_count += 1

        return even_count